ScrapeRepos.cjs

```diff
@@ -0,0 +1,243 @@
+const axios = require('axios');
+const cheerio = require('cheerio');
+const { time } = require('console');
+const showdown = require('showdown');
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+
+async function getUserInput() {
+  return new Promise(resolve => {
+    readline.question('From which timeframe do you want to get your data (0: daily; 1: weekly; 2: monthly): ', input => {
+      resolve(input);
+    });
+  });
+}
+
+/** Fetches the repo owner and name for each repo on the GitHub trending page
+ * @param userInput 0: daily; 1: weekly; 2: monthly
+ * @returns an array that stores alternatingly the owner and the name of each repo
+ */
+async function fetchRepos(userInput) {
+  try {
+    const timeModes = ['?since=daily', '?since=weekly', '?since=monthly'];
```

Comment:
Could you define the time modes as variables. All different time modes could be different value.

Reply:
In which scenario do you think would the time modes have a different value? I mean they don't change over time do they?

Reply:
When code becomes bigger it would hard to track which index represents which timeMode and those modes can be used by other modules therefore, they might be stored in common class.

Reply:
I will implement this in my next PR